doc/releases/changelog-dev.md

```diff
@@ -143,105 +150,118 @@
-  >>> print(params)
-  [-5.92774911 -4.26420843]
-  >>> print(cost(params))
-  0.43866366253270167
-  >>> max_iterations = 50
-  >>> opt = qml.SPSAOptimizer(maxiter=max_iterations)
-  >>> for _ in range(max_iterations):
-  ...     params, energy = opt.step_and_cost(cost, params)
-  >>> print(params)
-  [-6.21193761 -2.99360548]
-  >>> print(energy)
-  -1.1258709813834058
+  >>> x, y = np.array([0.4, 0.23], requires_grad=True)
+  >>> qml.gradients.param_shift(circuit, broadcast=True)(x, y)
+  (tensor(-0.38429095, requires_grad=True),
+   tensor(0.00899816, requires_grad=True))
   ```
 
-* Differentiable zero-noise-extrapolation error mitigation via ``qml.transforms.mitigate_with_zne`` with ``qml.transforms.fold_global`` and ``qml.transforms.poly_extrapolate``.
-  [(#2757)](https://github.com/PennyLaneAI/pennylane/pull/2757)
+  To illustrate the speedup, consider the following figure which shows, for a 
+  constant-depth circuit with Pauli rotations and controlled Pauli rotations, the 
+  time required to compute `qml.gradients.param_shift(circuit, broadcast=False)(params)`
+  ("No broadcasting") and `qml.gradients.param_shift(circuit, broadcast=True)(params)` 
+  ("Broadcasting") as a function of the number of qubits.
 
-  When using a noisy or real device, you can now create a differentiable mitigated qnode that internally executes folded circuits that increase the noise and extrapolating with a polynomial fit back to zero noise. There will be an accompanying demo on this, see [(PennyLaneAI/qml/529)](https://github.com/PennyLaneAI/qml/pull/529).
+  <img src="https://pennylane.readthedocs.io/en/latest/_images/default_qubit_native_broadcast_speedup.png" width=70%/>
 
-  ```python
-  # Describe noise
-  noise_gate = qml.DepolarizingChannel
-  noise_strength = 0.1
+  For transparency, the base code used for this example is given below:
 
-  # Load devices
-  dev_ideal = qml.device("default.mixed", wires=n_wires)
-  dev_noisy = qml.transforms.insert(noise_gate, noise_strength)(dev_ideal)
+  ```python
+  dev = qml.device("default.qubit", shots=None, wires=num_wires) 
+
+  def constant_depth_Pauli_rotations(param, wires):
+      [qml.Hadamard(w) for w in wires]
+      [qml.RX(p, w) for p, w in zip(param[0], wires)]
+      [qml.RY(p, w) for p, w in zip(param[1], wires)]
+      qml.broadcast(qml.CRX, wires=wires, pattern="ring", parameters=param[2])
+      [qml.RY(p, w) for p, w in zip(param[3], wires)]
+      [qml.RZ(p, w) for p, w in zip(param[4], wires)]
+      qml.broadcast(qml.CRX, wires=wires, pattern="ring", parameters=param[5])
+
+      return qml.probs(wires)
+
+  for num_wires in list(range(3, 13)):
+    dev = qml.device("default.qubit", wires=num_wires)
+    circuit = qml.QNode(constant_depth_Pauli_rotations, device=dev)
+    param = np.random.random(ansatz.shape_fn(num_wires))
+    
+    # time qml.gradients.param_shift(circuit, broadcast=False)(param) 
+    # time qml.gradients.param_shift(circuit, broadcast=True)(param)
+  ```
```

Comment:
Thoughts on this @rmoyard ? Also, not sure how to go about uploading figures here...

Reply:
I think we should be more concise here, we don't necessarily need to add the code.

Reply:
I'm fine with this. Can we still show the figure?

Reply:
Yes I think the figure totally make sense 👍